pkg/xds/server/callbacks/dataplane_status_sink.go

```diff
@@ -20,29 +20,35 @@ type DataplaneInsightSink interface {
 }
 
 type DataplaneInsightStore interface {
```

Comment:
So this later should become `ProxyInsightStore` or something like this?

Reply:
I don't know whether we would rename this if we add a new DP type. If we do, I agree that `ProxyInsightStore` is probably a better name.